src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -126,6 +118,64 @@ def make_parameter_file(
         json.dump(output, f)
 
 
+def translate_outputs(product_dir: Path, product_name: str):
+    """Translate ISCE outputs to a standard GTiff format with a UTM projection
+
+    Args:
+        product_dir: Path to the ISCE merge directory
+        product_name: Name of the product
+    """
+    ISCE2_DATASET = namedtuple('ISCE2_DATASET', ['name', 'suffix', 'band'])
+    datasets = [
+        ISCE2_DATASET('filt_topophase.unw.geo', 'unw_phase', 2),
+        ISCE2_DATASET('phsig.cor.geo', 'corr', 1),
+        ISCE2_DATASET('z.rdr.full.geo', 'dem', 1),
+        ISCE2_DATASET('filt_topophase.unw.conncomp.geo', 'conncomp', 1),
+        ISCE2_DATASET('filt_topophase.flat.geo', 'wrapped_phase', 1),
```

Comment:
Camel case is the convention for class names.
```suggestion
    ISCE2Dataset = namedtuple('ISCE2_DATASET', ['name', 'suffix', 'band'])
    datasets = [
        ISCE2Dataset('filt_topophase.unw.geo', 'unw_phase', 2),
        ISCE2Dataset('phsig.cor.geo', 'corr', 1),
        ISCE2Dataset('z.rdr.full.geo', 'dem', 1),
        ISCE2Dataset('filt_topophase.unw.conncomp.geo', 'conncomp', 1),
        ISCE2Dataset('filt_topophase.flat.geo', 'wrapped_phase', 1),
```